### ISSUE TITLE
Helm chart v1.7.0 clusterrole has insufficient perms

### ISSUE THREAD

**Issue by uberspot** [zalando/postgres-operator#1606]
- **Which image of the operator are you using?** 
registry.opensource.zalan.do/acid/postgres-operator:v1.7.0

- **Where do you run it - cloud or metal? Kubernetes or OpenShift?** 
Openshift 4.7
- **Are you running Postgres Operator in production?** 
yes
- **Type of issue?** 
 Bug report

```
{"cluster-name":"mynamespace/myapp-postgres-cluster","level":"error","msg":"could not create cluster: could not create master endpoint: could not create master endpoint: endpoints is forbidden: User \"system:serviceaccount:mynamespace:postgresql-operator-postgres-operator\" cannot create resource \"endpoints\" in API group \"\" in the namespace \"mynamespace\"","pkg":"controller","time":"2021-09-01T16:18:48Z","worker":0}
```

Missing RBAC perms for cluster_operator when used with the following values:

```
  values:
    enableJsonLogging: true
    configGeneral:
      kubernetes_use_configmaps: true
```

Most likely because of this if case: https://github.com/zalando/postgres-operator/blob/master/charts/postgres-operator/templates/clusterrole.yaml#L60

**Comment by FxKu** [comment]
I think, it's because of a bug that was fixed with #1603 . Bool fields were omitted hence `kubernetes_use_configmaps` might not have ended up in the configmap and the internal default to use endpoints was chosen.

**Comment by FxKu** [comment]
I have updated the helm chart for 1.7.0 in the gh-pages release brunch. Please, try again.
